src/app/core/model/EmoloyeeListResponse.ts

```diff
@@ -0,0 +1,9 @@
+import { ListResponseInterface } from './ListResponseInterface';
+import { Employee } from './Employee';
+
+export class EmployeesListResponse implements ListResponseInterface {
```

Comment:
use this model https://github.com/geekhub-php/theatre-front/blob/develop/src/app/core/model/employee/EmployeesListResponse.ts